conductor/example-config/holo.toml

```diff
@@ -0,0 +1,65 @@
+bridges = []
+signing_service_uri = "http://localhost:8888"
+
+[[agents]]
+id = "test agent 1"
+key_file = "holo_tester.key"
+name = "Holo Tester 1"
+public_address = "HoloTester1-----------------------------------------------------------------------AAACZp4xHB"
+holo_remote_key = true
```

Comment:
Remote keys could potentially be a use case outside of Holo. Does it make sense to start thinking about it as such, and not use 'holo' in the name? Also isn't key_file irrelevant now?